config/services.yaml

```diff
@@ -4,6 +4,8 @@
 # Put parameters here that don't need to change on each machine where the app is deployed
 # https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
+    app.pagination_default_limit: 30
+    app.pagination_default_start: 1
```

Comment:
I don't think this value is really useful

Reply:
you talk about the ```app.pagination_default_start```, ritgh ?

Reply:
yes :+1: